controls/pcidss_4_ocp4.yml

```diff
@@ -2907,8 +2929,10 @@ controls:
         Stored activity records cannot be modified without an alert being generated.
       levels:
         - base
-      status: pending
-      rules: []
+      status: partial
+      rules:
+        # TODO: Add FIO config to allow /var/log/... to extend in size but monitor perms.
```

Comment:
Good idea. We could track this as an issue or bug against FIO directly.

Reply:
This was actually extracted from the 3.2.1 version, :grimacing: